config/locales/en.yml

```diff
@@ -501,7 +509,7 @@ en:
       "npq-early-years-leadership": "NPQEYL"
       "npq-leading-literacy": "NPQLL"
       "npq-leading-primary-mathematics": "NPQLPM"
-      "npq-senco": "NPQS"
+      "npq-senco": "NPQSENCO"
```

Comment:
I think this might be intentional, might be worth checking with the NPQ team and @TBenedyk ?